lib/hard-context-module.js

```diff
@@ -68,12 +68,17 @@ HardContextModule.prototype.readableIdentifier = function(requestShortener) {
 HardContextModule.prototype.build = function(options, compilation, resolver, fs, callback) {
   this.builtTime = this.cacheItem.builtTime;
   var cacheItem = this.cacheItem;
-  var state = {state: {imports: {}}};
-  this.dependencies = deserializeDependencies.dependencies.call(state, cacheItem.dependencies, this);
-  this.variables = deserializeDependencies.variables.call(state, cacheItem.variables, this);
-  deserializeDependencies.blocks.call(state, cacheItem.blocks, this);
-  // this.warnings = cacheItem.warnings.map(deserializeError(ModuleWarning, state), this);
-  // this.errors = cacheItem.errors.map(deserializeError(ModuleError, state), this);
```

Comment:
We're removing all usages of the `deserializeError` method. Can we also loose that method?